ghost/admin/app/components/posts/analytics.js

```diff
@@ -201,39 +201,19 @@ export default class Analytics extends Component {
     }
 
     updateLinkData(linksData) {
-        let updatedLinks;
-        if (this.links?.length) {
-            updatedLinks = this.links.map((link) => {
```

Comment:
@kevinansfield Do you know of any reason why this shouldn't be removed - why we would need to update the links "in place"?